app/models/chapter_section.rb

```diff
@@ -0,0 +1,2 @@
+class ChapterSection < Sequel::Model(:chapters_sections)
```

Comment:
do we need this class?

Reply:
@matthewford yep I'm using it in Rspec tests for linking Sections and Chapters.

```
  let!(:chapter) do
    ch = create(:chapter,
      goods_nomenclature_item_id: chapter_code,
      producline_suffix: "80",
      validity_start_date: Date.today
    )

    period = add_period(ch.goods_nomenclature_sid, chapter_code)
    add_description(period,
      ch.goods_nomenclature_sid,
      chapter_code,
      "Inorganic chemicals; organic or inorganic compounds of precious metals, of rare-earth metals, of radioactive elements or of isotopes"
    )

    create(:chapter_section,
      goods_nomenclature_sid: ch.goods_nomenclature_sid,
      section_id: section.id
    )

    ch
  end
```

Reply:
ok